docs/tutorials/running_on_lfrqa.md

```diff
@@ -0,0 +1,292 @@
+manifest["citation"] = ""
+manifest.drop(columns=["doc_id", "doc_text"], inplace=True)
+manifest.to_csv(
+    os.path.join(lfrqa_directory, "science_docs_for_paperqa", "manifest.csv"),
+    index=False,
+)
+```
+
+## Filter and Save Questions
+
+Finally, we filter the question set to ensure we only include questions that reference the selected documents:
+
+```python
+partial_questions = questions[
+    questions.gold_doc_ids.apply(
+        lambda ids: all(id < amount_of_docs_to_use for id in ids)
+    )
+]
+partial_questions.to_csv(
+    os.path.join(lfrqa_directory, "questions.csv"),
+    index=False,
+)
+```
+
+## Install paperqa
+
+```bash
+pip install paper-qa
+```
+
+## Index the documents
+
+Copy the following to a file and run it. Feel free to adjust the concurrency as you like.
+
+PaperQA builds the index every time a question is asked and new files are present. So running this will build the index for you.
+
+You don’t need any api keys for building the index, but you do need them to answer questions.
+
+This process is quick for small portions of the whole document’s dataset, but can take ~3hs for the whole of it.
+
+If you want to use the index we built with the whole dataset, you can get it here. Make sure that you have all the docs in the `data/rag-qa-benchmarking/lfrqa/science_docs_for_paperqa/files` folder so the indexing process doesn't delete the missing ones.
+
+```python
+from paperqa import Settings, ask
+from paperqa.settings import AgentSettings, IndexSettings, ParsingSettings
+
+settings = Settings(
+    agent=AgentSettings(
+        index=IndexSettings(
+            name="lfrqa_science_index",
+            paper_directory="data/rag-qa-benchmarking/lfrqa/science_docs_for_paperqa",
+            index_directory="data/rag-qa-benchmarking/lfrqa/science_docs_for_paperqa_index",
+            manifest_file="manifest.csv",
+            concurrency=10_000,
+            batch_size=10_000,
+        )
+    ),
+    parsing=ParsingSettings(
+        use_doc_details=False,
+        defer_embedding=True,
+    ),
+)
+answer_response = (
+    ask(
+        "$5^n+n$ is never prime?",
+        settings=settings,
+    ),
+)
+```
+
+After this runs, you will get an answer!
+
+## Benchmark!
+
+After you have built the index, you are ready to run the benchmark.
+
+Copy the following into a file `gradable.py` and run it. You can also use the parameter num_questions in `LFRQATaskDataset` so you can make quick tests.
+
+To run this, you will need to have the [`ldp`](https://github.com/Future-House/ldp) package installed.
+
+```python
+import pandas as pd
+import os
+import json
+import asyncio
+from ldp.agent import SimpleAgent
+from ldp.alg.callbacks import MeanMetricsCallback
+from ldp.alg.runners import Evaluator, EvaluatorConfig
+from paperqa import Settings
+from paperqa.settings import AgentSettings, IndexSettings
+from paperqa.agents.task import LFRQATaskDataset, LFRQAQuestion
+from typing import List
```

Comment:
Two comments:
- Can you `isort` this (it's ported `ruff`)?
- Why do you have `from typing import List`, this is Python 3.11+ so it's not necessary